### ISSUE TITLE
Issue installing jaxlib 0.3.20 as specified in notebook-requirements.txt

### ISSUE THREAD

**Issue by tgjohnst** [aws-solutions-library-samples/aws-batch-arch-for-protein-folding#21]
Hi @brianloyal ! We have had this solution running and working well for a few months and it helped us bootstrap a lot of protein design work! However, we recently restarted the notebook instance and went to reinstall requirements and are receiving the following error. It looks like jaxlib 0.3.20 is no longer hosted in pypi (in fact, no jaxlib versions prior to 4.0) and thus package installation fails; batchfold installation also then fails and we get the `ModuleNotFoundError: No module named 'batchfold'` error reported by other users in issue #19 

Can you advise how we might get around this? Are later versions of jaxlib that might be compatible, and/or is there a way you'd suggest to install it from source?

Thanks in advance!

```
%pip install -q -U -r notebook-requirements.txt

ERROR: Ignored the following yanked versions: 0.4.0, 0.4.15
ERROR: Could not find a version that satisfies the requirement jaxlib==0.3.20 (from versions: 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.6, 0.4.7, 0.4.9, 0.4.10, 0.4.11, 0.4.12, 0.4.13, 0.4.14, 0.4.16, 0.4.17, 0.4.18, 0.4.19, 0.4.20, 0.4.21, 0.4.22, 0.4.23)
ERROR: No matching distribution found for jaxlib==0.3.20
Note: you may need to restart the kernel to use updated packages.
```

**Comment by kjogr12** [comment]
Hi @tgjohnst !
I've encountered the same error as well. I'm not familiar with sagemaker, so I might be mistaken, but it seems that the jax/jaxlib being installed here is not actually being used. It seems to me that the actual protein-related calculations are being done using Docker. As a test, I deleted jaxlib and installed jax==0.4.14, and then the sample notebook ran well.

Thank you.

**Comment by tgjohnst** [comment]
@kjogr12 this solution worked for us as well! Thank you so much for the suggestion.

**Comment by brianloyal** [comment]
Resolved with v1.12.6 🚀
